tests/components/zwave_js/test_discovery.py

```diff
@@ -134,6 +134,28 @@ async def test_merten_507801(
     assert state
 
 
+async def test_shelly_001p10_disabled_entities(
```

Comment:
Please also test that the entity for the other endpoint is created as expected and gets a state in the state machine.

Reply:
What do you mean exactly with "get the state in the state machine". This is my first pull request and also first touch with home assistant development. I tested it with my physical device here as a custom component, everything is working as expected. Is it enough to get the state from the first entity with 

`state = hass.states.get("cover.wave_shutter")`
`assert state`

like in the other test functions.

Reply:
Yes, exactly! 👍